ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXHeightMapVisualizer.java

```diff
@@ -61,28 +61,31 @@ public class RDXHeightMapVisualizer extends RDXVisualizer
 
    private int compressedBufferDefaultSize = 100000;
 
-   private ROS2Heartbeat activeHeartbeat;
-
    private float pixelScalingFactor = 10000.0f;
    private boolean heightMapMessageGenerated = false;
 
    public RDXHeightMapVisualizer()
    {
-      super("Height Map");
+      this("Height Map");
+   }
+
+   public RDXHeightMapVisualizer(String title)
+   {
+      super(title);
 
-      executorService = Executors.newFixedThreadPool(4);
+      executorService = MissingThreadTools.newSingleThreadExecutor("Height Map Visualizer Subscription", true, 1);
```

Comment:
This was used to stop a bunch of execution rejected exceptions and is the better strategy anyways.

Reply:
Doesn't this reduce the number of threads for this executor from 4 to 1?

Reply:
Yes, which is what you want, right? What's the point in having more than one here?

Reply:
It might be good to leave it how it was, is there a problem with having 4? Does that cause problems that make us want to use 1?

Reply:
Yes, there were rejected execution exceptions blowing up the console